roll/__init__.py

```diff
@@ -270,7 +275,7 @@ def __init__(self):
     def factory(self):
         return self.Protocol(self)
 
-    def route(self, path: str, methods: list=None, **extras):
+    def route(self, path: str, methods: list=None, **extras: dict):
```

Comment:
Can `**extras` be something else than a dict?

Reply:
Good question, I guess no, it's more as a matter of consistency in my experiment to annotate all parameters but it's quite useless… (for now!)